README.md

```diff
@@ -35,7 +35,21 @@ For **BREAKING CHANGES** Type a brief description of the breaking change when as
 
 ## Consuming
 
-- `> npm install @selectquotelabs/sqform`
+This is a Typescript library and therefore relies on Typescript types from libraries other than this one. The libraries who's types SQForm relies on are marked as a [peer dependency](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#peerdependencies) if the library is also required for non-type imports. Otherwise, they're marked as an [optional dependency](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#optionaldependencies).
+
+By default in NPM v7 and above peer dependencies will be installed automatically. You can check your npm version using `npm --version` to determine the version you are using. If your version is not at least v7 you must install the peer dependencies manually.
```

Comment:
For any devs reading this, please ensure you're using, NPM v6. v7 is quite the breaking change and we've run into problems in every project we've tried it out on